src/main/java/uk/gov/hmcts/darts/cases/exception/CaseApiError.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.hmcts.darts.cases.exception;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import uk.gov.hmcts.darts.common.exception.DartsApiError;
+
+@Getter
+@RequiredArgsConstructor
+public enum CaseApiError implements DartsApiError {
+
+    COURTHOUSE_PROVIDED_DOES_NOT_EXIST(
+        "100",
+        HttpStatus.BAD_REQUEST,
+        "Provided courthouse does not exist"
+    ),
+
+    COURTROOM_PROVIDED_DOES_NOT_EXIST(
+        "101",
+        HttpStatus.BAD_REQUEST,
+        "Provided courtroom does not exist"
+    ),
+    HEARING_MISMATCH(
```

Comment:
When is this error used?

Reply:
Not required anymore, it was added when we our design could potentially not distinguish between multiple hearings.